syntax = "proto3";

message ArchProto {
  float accuracy = 1;
  int32 id = 2;
  repeated  CellProto  cells = 3;
}

message CellProto {
  repeated  NodeProto  nodes = 1;
}

message NodeProto {
  repeated  string  content = 1;
}